contrib/pyln-proto/pyln/proto/wire.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, rawkey):
             raise ValueError(f"rawkey must be 32-byte long. {len(rawkey)} received")
 
         self.rawkey = rawkey
-        self.key = coincurve.PrivateKey.from_int(int(hexlify(rawkey), base=16))
+        self.key = coincurve.PrivateKey(rawkey)
```

Comment:
this commit is unnecessary, since it reverts a change in the prior commit in this same patchset. You should fold these into the same commit.